docs/.vuepress/utils/constants.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Shared constants for analytics implementation
+ */
+
+export const GA_MEASUREMENT_ID = 'G-05XJ24KPYH';
+export const CONSENT_KEY = 'ga_consent_given';
```

Comment:
```suggestion
export const LOCALHOST_CONSENT_KEY = 'ga_consent_given';
```

Could add here as well all the custom events to make it easier to refer to them

Reply:
I added the custom events.  Didn't change to LocalHost_consent_key though.